cinema/models.py

```diff
@@ -50,8 +50,8 @@ class Movie(models.Model):
     title = models.CharField(max_length=255)
     description = models.TextField()
     duration = models.IntegerField()
-    genres = models.ManyToManyField(Genre)
-    actors = models.ManyToManyField(Actor)
+    genres = models.ManyToManyField(Genre, blank=True)
+    actors = models.ManyToManyField(Actor, blank=True)
```

Comment:
The `genres` and `actors` fields in the `Movie` model are defined as `ManyToManyField` with `blank=True`. While this allows for movies without genres or actors, consider whether this aligns with your application's requirements. If a movie should always have at least one genre or actor, you might want to remove `blank=True`.